cmd/earth/main.go

```diff
@@ -1450,6 +1456,36 @@ func (app *earthApp) newBuildkitdClient(ctx context.Context, opts ...client.Clie
 	return bkClient, nil
 }
 
+func (app *earthApp) collectAnalytics(exitCode int, realtime time.Duration) {
+	if app.cfg.Global.DisableAnalytics {
+		return
+	}
+	installID, err := getInstallID()
+	if err != nil {
+		installID = "unknown"
+	}
```

Comment:
It seems that most CIs have the env var `CI` set to true:

* https://circleci.com/docs/2.0/env-vars/
* https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables
* https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
* https://buildkite.com/docs/pipelines/environment-variables#bk-env-vars-ci

Jenkins seems like the odd one out, but we could perhaps detect `BUILD_ID` there: https://wiki.jenkins.io/display/JENKINS/Building+a+software+project

Perhaps we can add a property like `"ci": "true"` when we detect a CI environment.

Reply:
that's a good idea, that way we'll avoid lots of unique installIDs since the uuid won't be saved between CI runs.

Reply:
Yes, exactly